swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+   def __init__(self, id = None, condition = 0, age = None):
+      self.id = uuid.uuid1().int if id is None else id
+      self.category = "Item"
+      self.condition = condition
+      self.age = age
+   
+   def get_category(self):
+      return self.category
+   
+   def __str__(self):
+      return f"An object of type Item with id {self.id}."
+   
+   def condition_description(self):
+      match(self.condition):
+         case 0.0:
+            return "It sucks!"
+         case 1.0:
+            return "Not worth it."
+         case 2.0:
+            return "This looks rough."
+         case 3.0:
+            return "Good but not great."
+         case 4.0:
+            return "Such a deal."
+         case 5.0:
+            return "You're in luck, this is a steal."
```

Comment:
This implementation works! It could be considered brittle (prone to breaking) since the data relationships are encoded here (mapping a number to a string).

What data structures do you know of that could capture such an associative relationship like this? How about a dictionary with its key/value pairs? 

```python
ITEM_CONDITIONS = {
    0: "It sucks!", 
    1: "Not worth it.",
    2: "This looks rough.", ... 
}

return ITEMS_CONDITIONS.get(self.condition)
```

Having a clear representation of the data can make code easier to read!